### PR TITLE
Mi 30 modularise analysis tab

### PR DESCRIPTION
**Pull request by JanionNevill**
Each of the tabs on the data analysis page has been split into a dedicated tab, as well as the sidebar containing the options.

### CODE REVIEW

**Review by BalancedmonkeY — CHANGES_REQUESTED** [review]
Code ran fine.

Suggested changes are only relating to roxygen comments

File: R/bayesian_analysis_panel.R [hunk 1]
```diff
@@ -0,0 +1,1350 @@
...
+  )
+}
+
+
+#' Module server for the bayesian analysis panel.
+#' 
+#' @param id ID of the module
+#' @param data Reactive containing data to analyse
+#' @param treatment_df Reactive containing data frame containing treatment IDs (Number) and names (Label)
+#' @param metaoutcome Reactive containing meta analysis outcome: "continuous" or "binary"
```
[the remark this answers is not in the document]
Comment (JanionNevill) [reply R/bayesian_analysis_panel.R:663]: Thanks, fixed.

File: R/bayesian_analysis_panel.R [hunk 2]
```diff
@@ -0,0 +1,1350 @@
...
+#' @param outcome_measure Reactive containing meta analysis outcome measure: "MD", "SMD", "OR, "RR", or "RD"
+#' @param continuous_outcome Reactive containing acronym of the continuous outcome:
+#'   "MD" for mean difference, or "SMD" for standardised mean difference
+#' @param binary_outcome Reactive containing acronym of the binary outcome:
+#'   "OR" for odds ratio, "RR" for risk ratio, or "RD" for risk difference
+#' @param model_effects Reactive containing model effects: either "random" or "fixed"
+#' @param exclusions Reactive containing names of studies excluded from the sensitivity analysis
+#' @param rank_option Reactive containing ranking option: "good" or "bad" depending on whether small values are desirable or not
+#' @param freq_all Reactive containing frequentist meta-analysis for the sensitivity analysis
+#' @param freq_sub Reactive containing frequentist meta-analysis
```
[the remark this answers is not in the document]
Comment (JanionNevill) [reply R/bayesian_analysis_panel.R:673]: Thanks, fixed.

File: R/data_analysis_options_panel.R [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  )
+}
+
+
+#' Module server for the data analysis options panel
+#' 
+#' @param id ID of the module
+#' @param data Reactive containing data to analyse
+#' @param is_default_data Reactive containing TRUE if data is an example dataset, loaded by default
+#' @param metaoutcome Reactive containing meta analysis outcome: "continuous" or "binary"
```
[the remark this answers is not in the document]
Comment (JanionNevill) [reply R/data_analysis_options_panel.R:68]: Done.

[the remark this answers is not in the document]
Comment (JanionNevill) [reply R/data_analysis_options_panel.R:68]: Done.

**Review by BalancedmonkeY — APPROVED** [review]

**Pull request merged**